app/static/pursuit.less

```diff
@@ -486,6 +574,11 @@ ol li {
   border-top: 1px solid darken(@background, 20%);
   border-bottom: 1px solid darken(@background, 20%);
   padding: 0;
+
+  @media (prefers-color-scheme: dark) {
+    border-top: 1px solid lighten(@dark_background, 20%);
+    border-bottom: 1px solid lighten(@dark_background, 20%);
```

Comment:
Shouldn't these be `border-top-color` and `border-bottom-color`?

Reply:
Good catch! Yes, these should probably specify just the color. I've changed this accordingly.